examples/bridge-app/asr/subdevice/SubDeviceManager.cpp

```diff
@@ -48,7 +48,7 @@ static EndpointId gFirstDynamicEndpointId;
 static SubDevice * gSubDevices[CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT]; // number of dynamic endpoints count
 
 int AddDeviceEndpoint(SubDevice * dev, EmberAfEndpointType * ep, const Span<const EmberAfDeviceType> & deviceTypeList,
-                      const Span<DataVersion> & dataVersionStorage, chip::EndpointId parentEndpointId)
+                      Span<DataVersion> && dataVersionStorage, chip::EndpointId parentEndpointId)
```

Comment:
Why is this an rvalue?  Doesn't that allow caller to pass a temporary?  `dataVersionStorage` can't be a temporary...